build/depends.py

```diff
@@ -290,19 +280,14 @@ def configure(self, build, conf):
 
         # Enable Qt include paths
         if build.platform_is_linux:
-            if qt5 and not conf.CheckForPKG('Qt5Core', '5.0'):
+            if not conf.CheckForPKG('Qt5Core', '5.0'):
                 raise Exception('Qt >= 5.0 not found')
```

Comment:
What's our actually required minimum version now? I think at least 5.6 or 5.5?

Reply:
Hm, I'm not sure. I believe we still build successfully on Trusty which uses [Qt 5.2](https://packages.ubuntu.com/trusty/libs/qt5-default).